geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHash.java

```diff
@@ -105,7 +104,7 @@ public RedisHash(List<byte[]> fieldsToSet) {
   }
 
   public RedisHash() {
-    sizeInBytes = BASE_REDIS_HASH_OVERHEAD;
+    // For serialization only!
```

Comment:
If this is used only for serialization:
1. It probably needs an annotation to suppress the unused compile warning.
2. Should be document in the Javadocs so that its visible to the consumer of the class without looking at the source what the intent for the method is.

Reply:
No compile warnings, so we're good there. I added the javadocs.